creator-node/src/diskManager.ts

```diff
@@ -79,26 +79,13 @@ export async function getTmpTrackUploadArtifactsPath() {
   return dirPath
 }
 
-/**
- * Construct the path to a file or directory given a CID
- *
- * eg. if you have a file CID `Qmabcxyz`, use this function to get the path /file_storage/files/cxy/Qmabcxyz
- * eg. if you have a dir CID `Qmdir123`, use this function to get the path /file_storage/files/r12/Qmdir123/
- * Use `computeFilePathInDir` if you want to get the path for a file inside a directory.
- *
- * @dev Returns a path with the three characters before the last character
- *      eg QmYfSQCgCwhxwYcdEwCkFJHicDe6rzCAb7AtLz3GrHmuU6 will be eg /file_storage/muU/QmYfSQCgCwhxwYcdEwCkFJHicDe6rzCAb7AtLz3GrHmuU6
- * @param {String} cid file system destination, either filename or directory
- */
-export async function computeFilePath(cid: string, ensurePathExists = true) {
+function _getParentDirPathForCid(cid: string) {
```

Comment:
can we call this something besides parentDir because i think that's overloaded with dirCID. i was confused for a min looking at that. maybe something like _getStorageLocationForCID or something?

Reply:
changed 👍